src/blackjack/card-value.js

```diff
@@ -1,3 +1,28 @@
+
+
+
 export default function cardValue(cardName) {
 
-}
+  if (cardName === '5') {
```

Comment:
We'll double check, but if `cardName` isn't ace, jack, king, or queen you should be able to just turn the string into a number and :shipit: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt